person/src/Dancer.java

```diff
@@ -0,0 +1,55 @@
+
+
+/**
+ * Class Dancer
+ */
+public class Dancer extends Person {
```

Comment:
@tatevikb jan did you think about separation of actions from people?  In that case you will have more abstraction